.github/workflows/rc.yml

```
@@ -0,0 +1,640 @@
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git tag -a "${RC_TAG}" -m "Release candidate ${RC_TAG}"
+          git push origin "${RC_TAG}"
+
+      - name: Release summary
+        env:
+          VERSION: ${{ needs.prepare-release.outputs.version }}
+          RC_TAG: ${{ needs.prepare-release.outputs.rc_tag }}
+          RC_NUMBER: ${{ needs.prepare-release.outputs.rc_number }}
+          PEP440_VERSION: ${{ needs.prepare-release.outputs.pep440_version }}
+          WHEEL_FILENAME: ${{ needs.prepare-release.outputs.wheel_filename }}
+          GPU_TEST_RESULT: ${{ needs.gpu-e2e-tests.result }}
```

actionlint (1.7.12)

[error] property "gpu-e2e-tests" is not defined in object type {approve-release: {outputs: {}; result: string}; prepare-release: {outputs: {pep440_version: string; rc_number: string; rc_tag: string; version: string; wheel_filename: string}; result: string}}

(expression)